### DISCUSSION TITLE
Verifying xml string content on windows (dev) and linux (ci/cd): failing

### DISCUSSION THREAD

**Discussion by ITaluone** [VerifyTests/Verify#821] [Q&A]
Hi 

Have a quick question: I am trying to verify xml content. On windows everything was green, so I checked all in.
But my CI/CD pipeline (which runs on debian) thinks the verified and received are not equal. Why?

Possible line endigs? But how to ignore them?

**Comment by ITaluone** [comment]
I have possibly found the issue, but don't know how to fix it :)

Decimal values do not have decimal points on linux (eg. the decimal value on windows is 123.45 and on linux it becomes 12345)

How can I fix this?

**Reply by SimonCropp** [reply]
submit a failing unit test as a pull request
